### ISSUE TITLE
Revisit the scaling of stateless + MUST map 5-tuple

### ISSUE THREAD

**Issue by mstojens** [furry13/v6ops-464xlat-enable#41]
Heavy WG discussion here that requires thinking through

**Comment by mstojens** [comment]
Pretty sure this is covered from multiple directions by other edits. Do you think there's something we missed here, @furry13 ? Base of course on the meticulous level of detail I provided here ;)

**Comment by furry13** [comment]
Yeah, I guess we sorted this out (we'll see what the WG thinks though)
